app/controllers/users_controller.rb

```diff
@@ -0,0 +1,23 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
```

Comment:
You'll have to restrict signed in users from hitting this action. Maybe do `redirect_to home_path and return if logged_in?`. The `return` makes sure the `@user` instance var is not hit; this will prevent any unnecessary instance var leaks through your app.